### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jotes**
Hi, Pardon me, but this looks like a typo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Shortcuts for wasm
 
 | wah       | wasm                                      |
 |---------------|-------------------------------------------|
-| `(0 = 1)`     | `(set_local 0 (f64.const 0))`             |
+| `(0 = 1)`     | `(set_local 0 (f64.const 1))`             |
```
Comment (jayphelps) [inline README.md]: It's actually `(set_local 0 (i32.const 1))` as wah will only use f64 for literals with a decimal.

**Comment by jotes** [comment]
@jayphelps could you look again? thanks!

**Comment by jayphelps** [comment]
@jotes LGTM but I don't actually work here 😆  I just was driving by and noticed the README mistake too, saw your PR and wanted to clarify the error in it.

@tmcw will have to check it out. 🚀

**Comment by tmcw** [comment]
🙏 Thanks @jotes!

**Pull request merged**